src/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -21,7 +21,7 @@ export class NavigationTabs extends React.Component<Props> {
     const { exact } = options
 
     return (
-      <RouteTab to={to} exact={exact}>
+      <RouteTab to={to} exact={exact} preload={false}>
```

Comment:
If we set this to `false` it will fall back to `<Link>` (from `found`) under the hood, and skip all preload behavior. For routes that depend on `prepareVariables` this is a must.